ddtrace/contrib/pymongo/parse.py

```diff
@@ -79,14 +79,12 @@ def parse_msg(msg_bytes):
         log.debug('unknown op code: %s', op_code)
         return None
 
+    # Only parse query messages
+    # NOTE[matt] inserts, updates and queries can all use this opcode
     db = None
     coll = None
-
-    offset = header_struct.size
-    cmd = None
     if op == 'query':
-        # NOTE[matt] inserts, updates and queries can all use this opcode
-
+        offset = header_struct.size
```

Comment:
That looked fine as it was, why moving it in each case?

Reply:
good catch, I was testing a bunch of approaches so this just happened to end up this way, I'll move back.